### ISSUE TITLE
Dataset Splitting

### ISSUE THREAD

**Issue by Jiayue-Yuan** [wjrzm/MotionPRO#5]
Hi, I have a small concern about the data path configuration in config/task/imagepressure2smpl.yaml.
The train_data_dir is set to /data1/shenghao/MotionPRO/, while the eval_data_dir is /data1/shenghao/MotionPRO/0729/csy/. As the eval path is a subfolder of the train root path, it may cause the training set to include the validation set samples accidentally.
Just want to check if this is an intended setting or a potential path configuration issue. Thanks a lot!

**Comment by wjrzm** [comment]
This is a typo; our training set and test set have no overlap.
